src/main.ts

```diff
@@ -9,6 +9,7 @@ import { NotionImporter } from './formats/notion';
 import { OneNoteImporter } from './formats/onenote';
 import { RoamJSONImporter } from './formats/roam-json';
 import { truncateText } from './util';
+import { NodePickedFile } from 'filesystem';
```

Comment:
Can we revert the changes to this file?

Reply:
The current option for debugging that's commented out is outdated—I'll revert these changes but I'd recommend updating the commented-out debugging option for developers